### ISSUE TITLE
Cannot import essentia.standard from ipython --pylab 

### ISSUE THREAD

**Issue by edufonseca** [MTG/essentia#500]
After begining an ipython session **with --pylab**, I can do: 
```
import essentia
import essentia.streaming
```
with no problem. However, when I try to:

`import essentia.standard`

I get:

```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-3-35df285b90a1> in <module>()
----> 1 import essentia.standard

/usr/local/lib/python2.7/dist-packages/essentia/standard.py in <module>()
    116         _create_essentia_class(name, moduleName)
    117 
--> 118 _reloadAlgorithms()
    119 
    120 

/usr/local/lib/python2.7/dist-packages/essentia/standard.py in _reloadAlgorithms(moduleName)
    114 def _reloadAlgorithms(moduleName = __name__):
    115     for name in _essentia.keys():
--> 116         _create_essentia_class(name, moduleName)
    117 
    118 _reloadAlgorithms()

/usr/local/lib/python2.7/dist-packages/essentia/standard.py in _create_essentia_class(name, moduleName)
     27     essentia.log.debug(essentia.EPython, 'Creating essentia.standard class: %s' % name)
     28 
---> 29     _algoInstance = _essentia.Algorithm(name)
     30     _algoDoc = _algoInstance.getDoc()
     31     _algoStruct = _algoInstance.getStruct()

ValueError: ERROR: Algorithm LoudnessEBUR128 could not be configured using default values.
       If it doesn't make sense for an algorithm to be configured with
       default values, then it should still be able to be instantiated, and
       it is your responsibility to keep track of the fact that it should
       currently be impossible to call it (for example, by checking if the state
       is valid upon entering the process() method).

Parameter hopSize = 0.10000000149 is not within specified range: (0,0.1]
```

**Note that no error arises from an ipython session WITHOUT --pylab.**

**Comment by x75** [comment]
also seeing this issue, both in ipython and when compiling code on the fly with ast

I changed the hopSize defaults in src/algorithms/temporal/loudnessebur128.h to 0.05 but somehow the Range get's confused, see final error msg below.

```
Traceback (most recent call last):
  File "/home/src/QK/smp_graphs/smp_graphs/common.py", line 246, in get_config_raw_from_string
    exec(code, global_vars, local_vars)
  File "<ast>", line 33, in <module>
  File "/home/src/QK/smp_graphs/smp_graphs/block_meas_essentia.py", line 18, in <module>
    import essentia.standard as estd
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 136, in <module>
    _reloadAlgorithms()
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 134, in _reloadAlgorithms
    _create_essentia_class(name, moduleName)
  File "/usr/local/lib/python2.7/dist-packages/essentia/standard.py", line 30, in _create_essentia_class
    _algoInstance = _essentia.Algorithm(name)
ValueError: ERROR: Algorithm LoudnessEBUR128 could not be configured using default values.
       If it doesn't make sense for an algorithm to be configured with
       default values, then it should still be able to be instantiated, and
       it is your responsibility to keep track of the fact that it should
       currently be impossible to call it (for example, by checking if the state
       is valid upon entering the process() method).

Parameter hopSize = 0.0500000007451 is not within specified range: (0,0.1]
```

Current workaround is 

```
diff --git a/src/python/essentia/standard.py b/src/python/essentia/standard.py
index e9f4e33..ea29573 100644
--- a/src/python/essentia/standard.py
+++ b/src/python/essentia/standard.py
@@ -131,6 +131,7 @@ def _create_essentia_class(name, moduleName = __name__):
 # load all classes into python
 def _reloadAlgorithms(moduleName = __name__):
     for name in _essentia.keys():
+       if name == 'LoudnessEBUR128': continue
         _create_essentia_class(name, moduleName)
 
 _reloadAlgorithms()
```

**Comment by dbogdanov** [comment]
@x75, @edufonseca  Can you still reproduce that with the latest master? Works fine for me.

**Comment by ech2** [comment]
Works fine for me.

```
$ ipython --pylab
Python 3.6.4 (default, Jan  5 2018, 02:35:40)
Type 'copyright', 'credits' or 'license' for more information
IPython 6.3.1 -- An enhanced Interactive Python. Type '?' for help.
Using matplotlib backend: Qt5Agg

In [1]: qt5ct: using qt5ct plugin
In [1]:

In [1]: import essentia.standard
```

**Comment by dbogdanov** [comment]
Cannot reproduce this issue using the latest master. Closing this issue.

**Comment by seffka** [comment]
It's driving me mad.
Reappear with essentia 2.1b5.dev707, Ubuntu 18.04.2 LTS.
It depends on the orer of imports, e.g.:
`import essentia.standard as ess`
works as the first statement.
Fails after, e.g.,  matplotlib:

`In [1]: import matplotlib.pyplot as plt 
   ...:                                                                                                                               

In [2]: import essentia.standard as ess                                                                                               
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-2-deb42d8a0bff> in <module>
----> 1 import essentia.standard as ess

~/.local/lib/python3.6/site-packages/essentia/standard.py in <module>
    134         _create_essentia_class(name, moduleName)
    135 
--> 136 _reloadAlgorithms()
    137 
    138 

~/.local/lib/python3.6/site-packages/essentia/standard.py in _reloadAlgorithms(moduleName)
    132 def _reloadAlgorithms(moduleName = __name__):
    133     for name in _essentia.keys():
--> 134         _create_essentia_class(name, moduleName)
    135 
    136 _reloadAlgorithms()

~/.local/lib/python3.6/site-packages/essentia/standard.py in _create_essentia_class(name, moduleName)
     28     essentia.log.debug(essentia.EPython, 'Creating essentia.standard class: %s' % name)
     29 
---> 30     _algoInstance = _essentia.Algorithm(name)
     31     _algoDoc = _algoInstance.getDoc()
     32     _algoStruct = _algoInstance.getStruct()

ValueError: ERROR: Algorithm LoudnessEBUR128 could not be configured using default values.
       If it doesn't make sense for an algorithm to be configured with
       default values, then it should still be able to be instantiated, and
       it is your responsibility to keep track of the fact that it should
       currently be impossible to call it (for example, by checking if the state
       is valid upon entering the process() method).

Parameter hopSize = 0.10000000149 is not within specified range: (0,0.1]
`

**Comment by nullmightybofo** [comment]
> It's driving me mad.
> Reappear with essentia 2.1b5.dev707, Ubuntu 18.04.2 LTS.
> It depends on the orer of imports, e.g.:
> `import essentia.standard as ess`
> works as the first statement.
> Fails after, e.g., matplotlib:
> 
> `In [1]: import matplotlib.pyplot as plt
> ...:
> ## In [2]: import essentia.standard as ess
> 
> ValueError Traceback (most recent call last)
> in
> ----> 1 import essentia.standard as ess
> 
> ~/.local/lib/python3.6/site-packages/essentia/standard.py in
> 134 _create_essentia_class(name, moduleName)
> 135
> --> 136 _reloadAlgorithms()
> 137
> 138
> 
> ~/.local/lib/python3.6/site-packages/essentia/standard.py in _reloadAlgorithms(moduleName)
> 132 def _reloadAlgorithms(moduleName = **name**):
> 133 for name in _essentia.keys():
> --> 134 _create_essentia_class(name, moduleName)
> 135
> 136 _reloadAlgorithms()
> 
> ~/.local/lib/python3.6/site-packages/essentia/standard.py in _create_essentia_class(name, moduleName)
> 28 essentia.log.debug(essentia.EPython, 'Creating essentia.standard class: %s' % name)
> 29
> ---> 30 _algoInstance = _essentia.Algorithm(name)
> 31 _algoDoc = _algoInstance.getDoc()
> 32 _algoStruct = _algoInstance.getStruct()
> 
> ValueError: ERROR: Algorithm LoudnessEBUR128 could not be configured using default values.
> If it doesn't make sense for an algorithm to be configured with
> default values, then it should still be able to be instantiated, and
> it is your responsibility to keep track of the fact that it should
> currently be impossible to call it (for example, by checking if the state
> is valid upon entering the process() method).
> 
> Parameter hopSize = 0.10000000149 is not within specified range: (0,0.1]
> `

Having the same problem. Importing essentia.standard on top (first import) works, otherwise not. I
 Essentia '2.1-beta5-dev' (via pip, under python3), Ubuntu  18.04.1.

**Comment by nullmightybofo** [comment]
Still same problem with '2.1-beta6-dev' on Ubuntu 18.04.1.

If essentia.standard not imported as first package, this error occurs.

**Comment by dbogdanov** [comment]
This should be fixed in c3ec967bce74747b5f99e75e51f636d622c64105. Please report again on any further issues.
